src/components/VisualPickerOption/index.js

```diff
@@ -108,6 +108,7 @@ VisualPickerOption.propTypes = {
      * @ignore
      */
     children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.object]),
+    ariaDescribedby: PropTypes.string,
```

Comment:
this is already fixed, this is not a prop passed directly to the VisualPickerOption but a prop that comes in context from parent component (VisualPicker), since it is only internal we should not add it to the proptypes in order to not get it exposed as documentation for consumers